src/components/renderer/LookupNotification.tsx

```diff
@@ -150,9 +155,17 @@ function LookupNotificationComponent({
           }
         }
 
+        const clonedElements = _cloneDeep(definition.elements)
+
+        formElementsService.forEachFormElement(clonedElements, (el) => {
+          if (el.type === 'page' && el.id === pageId) {
+            el.elements = allElements
+          }
+        })
```

Comment:
This seems very _sledge hammer to nail_ as an approach, could you please walk me through this change.

Reply:
Dont we just need to remove line 95 in `src/hooks/useFormElementOptions.ts` to solve this problem?

Reply:
or change the validation to not allow empty strings by setting `allowEmpty` to `false` in `src/services/form-validation.ts`.

https://validatejs.org/#validators-presence

Reply:
I started with allowEmpty, but there's a bigger problem that if the elements being filled are on another page then they won't go through the generateDefaultData function and then we're allowing all sorts of incorrect values onto the submission body some of which may be caught by the validator but I would guess a lot wouldn't.

Reply:
`generateDefaultData()` should not be validating the data that is applied to the submission. It should be setting default values for new elements that are being injected into the form. 

Either way, can we please create a new bug ticket to deal with this problem and get it prioritised/refined/planned? Avoiding scope creep was one of the outcomes of our last retro regarding P1s.

Reply:
Well, it's currently validating pre fill data.  Are you saying that validating pre fill is what should be split out and that is the separate story? Was this bug a P1? I'd also argue the solution isn't scope creep because the problem can only be reproduced if the elements being filled are on a different page, otherwise the validation works just fine.